### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
Modify readme file for a Code Review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+1. Setup
+2. Install
+3. Usage
+4. Tun tests
+5. Deployment
+
+#### Live Demo
+
+Link:<http://127.0.0.1:5500/index.html>
```
Comment (VanessaAoki) [inline README.md]: - [ ] In `.md` files, you should use `[link name](actual link)`, you can read more about it [here](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
-# microverse
\ No newline at end of file
+# Hello-Microverse
+
+## "Hello Microverse" project
+
+the purpose of this project is for mastering best practise, familiar with all the tools used such as linters, github, html and css
+
```
Comment (VanessaAoki) [inline README.md:7]: - [ ] To make your README look more professional, you should consider adding a screenshot of your project here.

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @ismailakinkunmi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._
#### Besides those comments you should also consider:
- [ ] Your Pull Request (PR) should have a short summary of the changes made in this branch.
- [ ] Your commits messages should be more meaningful.

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ismailakinkunmi) [reply README.md:6]: Considering this is my first PullRequest experience, is better to overlook some unnecessary things, Rome was not built in a day and this is not encouraging at all compared to my partner PR.

File: README.md (hunk 1 above)
Comment (ismailakinkunmi) [reply README.md:33]: This should just be a note for improvement.

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ismailakinkunmi ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly
- Descriptive Pull Request
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**